rules/windows/malware/av_relevant_files.yml

```diff
@@ -10,33 +10,35 @@ logsource:
     product: antivirus
 detection:
     selection:
-        FileName:
-            - 'C:\Windows\Temp\\*'
-            - 'C:\Temp\\*'
-            - '*\\Client\\*'
-            - 'C:\PerfLogs\\*'
-            - 'C:\Users\Public\\*'
-            - 'C:\Users\Default\\*'
-            - '*.ps1'
-            - '*.vbs'
-            - '*.bat'
-            - '*.chm'
-            - '*.xml'
-            - '*.txt'
-            - '*.jsp'
-            - '*.jspx'
-            - '*.asp'
-            - '*.aspx'
-            - '*.php'
-            - '*.war'
-            - '*.hta'
-            - '*.lnk'
-            - '*.scf'
-            - '*.sct'
-            - '*.vbe'
-            - '*.wsf'
-            - '*.wsh'
-    condition: selection
+        FileName|startswith:
+            - 'C:\Windows\Temp\\'
+            - 'C:\Temp\\'
+            - 'C:\PerfLogs\\'
+            - 'C:\Users\Public\\'
+            - 'C:\Users\Default\\'
+            - '*\\Client\\'
```

Comment:
original logic for `\Client\` was `contains`. you probably need to create another section with the

```
        FileName|contains: `\Client\`
```

or just use yaml syntax in single selection:


```
    selection:
        - FileName|startswith:
        - FileName|contains: 
        - Filename|endswith:
```

this also will fly (:

Reply:
The same situation of https://github.com/Neo23x0/sigma/pull/1193#discussion_r513121232

Reply:
Fixed :)